engine/apps/alerts/escalation_snapshot/escalation_snapshot_mixin.py

```diff
@@ -42,62 +42,6 @@ def build_raw_escalation_snapshot(self) -> dict:
         """
         Builds new escalation chain in a json serializable format (dict).
         Use this method to prepare escalation chain data for saving to alert group before start new escalation.
-
-        Example result:
-        {
-            'channel_filter_snapshot': {
-                'id': 1,
-                'notify_in_slack': True,
-                'str_for_clients': 'default',
-                'notify_in_telegram': True
-            },
-            'escalation_chain_snapshot': {
-                'id': 1,
-                'name': 'Test'
-            },
-            'escalation_policies_snapshots': [
-                {
-                    'id': 1,
-                    'step': 14,
-                    'order': 0,
-                    'to_time': None,
-                    'from_time': None,
-                    'num_alerts_in_window': None,
-                    'num_minutes_in_window': None,
-                    'wait_delay': None,
-                    'notify_schedule': None,
-                    'notify_to_group': None,
-                    'notify_to_team_members': None,
-                    'passed_last_time': None,
-                    'escalation_counter': 0,
-                    'last_notified_user': None,
-                    'custom_button_trigger': None,
```

Comment:
rather than just remove the `custom_button_trigger` field here from the docstring, I don't think we should be documenting large types like this in the docstring (this is what type hints are for 🙂)